build.gradle

```diff
@@ -954,6 +954,9 @@ project(':iceberg-snowflake') {
 
 project(':iceberg-open-api') {
   apply plugin: 'java-test-fixtures'
+  apply plugin: 'io.github.goooler.shadow'
```

Comment:
https://github.com/Goooler/shadow is an archived repo. Is the plugin still maintained?

Reply:
I used it because whole Iceberg project is using it. 

Just checked that very recently archived. I will raise a PR at Iceberg to use the respective plugin.